lucky_draw/config/config.exs

```diff
@@ -0,0 +1,31 @@
+# This file is responsible for configuring your application
+# and its dependencies with the aid of the Mix.Config module.
+#
+# This configuration file is loaded before any dependency and
+# is restricted to this project.
+
+# General application configuration
+use Mix.Config
+
+config :lucky_draw,
+  ecto_repos: [LuckyDraw.Repo]
+
+# Configures the endpoint
+config :lucky_draw, LuckyDrawWeb.Endpoint,
+  url: [host: "localhost"],
+  secret_key_base: "gi5oh8wq6yA/CI4mXwZtG1SvIXDQEFtwXdrnvDxVrQvqGGyUb5/WhxAWn2D3Zd3c",
```

Comment:
What's the usage of  **secret_key_base**  here? Injecting by system environment should be better.

Reply:
We need to generate a new `secret_key_base` when deploy to production according to the [document](https://hexdocs.pm/phoenix/deployment.html#handling-of-your-application-secrets).

I think this one here is for local development and the usage of this value can be found [here](https://hexdocs.pm/phoenix/Phoenix.Endpoint.html#content).